lib/modelMaterialsCommon.js

```diff
@@ -558,16 +558,18 @@ function generateTechnique(gltf, khrMaterialsCommon, lightParameters) {
     // Add shaders
     shaders[vertexShaderId] = {
         type: WebGLConstants.VERTEX_SHADER,
-        uri: 'data:,' + vertexShader,
         extras: {
-            source: vertexShader
+            _pipeline : {
+                source : vertexShader
+            }
```

Comment:
I made this change to fix a crash along the way since every `source` expects there to be `extras._pipeline.source`. I will have to handle Cesium compatibility for this https://github.com/AnalyticalGraphicsInc/OBJ2GLTF/issues/4#issuecomment-224075451